src/Sentry.AspNet/SentryAspNetOptionsExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static void AddAspNet(this SentryOptions options, RequestSize maxRequestB
 
             var eventProcessor = new SystemWebRequestEventProcessor(payloadExtractor, options);
 
-            options.DiagnosticLogger ??= new DebugDiagnosticLogger(options.DiagnosticLevel);
+            options.DiagnosticLogger ??= new TraceDiagnosticLogger(options.DiagnosticLevel);
```

Comment:
I'd replace the default logger with TraceDiagnosticLogger since it doesn't pollute the terminal screen if your application is console-based.

Reply:
Logging is opt-in so nothing gets poluted unless u turn it on.